deployment/script/build-CCNP.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -o errexit
+
+ccnp_url=https://github.com/intel/confidential-cloud-native-primitives.git
+device_plugin=docker.io/library/ccnp-device-plugin:0.1
+get_quote=docker.io/library/ccnp-quote-server:0.1
+measurement=docker.io/library/ccnp_measurement_server:0.1
+eventlog=ccnp_eventlog_server:0.1
+feature_url=https://github.com/kubernetes-sigs/node-feature-discovery.git
+
+add_proxy(){
+        sed -i "/FROM .* as [b,q]/I a\ENV https_proxy='$https_proxy'"  $1
+        sed -i "/FROM .* as [b,q]/I a\ENV http_proxy='$http_proxy'"  $1
+        sed -i "/FROM .* as [b,q]/I a\ENV HTTPS_PROXY='$https_proxy'"  $1
```

Comment:
For some test environment (non-Intel lab), there might be no proxy setting on the host, what will happen for such add_proxy usage?

Reply:
Actually, add ENV in Dockerfile is not a BKM, the better way is to use "docker build -e <ENV>" to pass the ENVs in build phase, in this way, the Dockerfile keeps the same in various of scenarios, only customers need to take care of those proxy issues due to their network scenarios. You can have a try about that.